### ISSUE TITLE
snoopy-install.sh fails test datasource_domain.sh

### ISSUE THREAD

**Issue by suedi** [a2o/snoopy#84]
Trying to install stable 2.4.3
Get a fail on tests/datasource/datasource_domain.sh test because

```bash
hostname -d
```
 returns "(none)"

```bash
VAL_SNOOPY=`$SNOOPY_TEST_DATASOURCE domain` 
```
also returns "(none)"

Here the test would pass but  VAL_SNOOPY gets corrected further down in code

```bash
if [ "$VAL_SNOOPY" == "(none)" ]; then
    VAL_SNOOPY=""
fi
```

which leads the test to fail

I commented out the correction and snoopy installed fine.

**Comment by bostjan** [comment]
I wondered how long it would take before someone reports some issue about this test :)

Anyhow, it seems that implementations of `hostname -d` vary a bit:
- some return empty strings
- others return "(none)" - as you have noted

I changed the test to account for this, it will be included in next release.

Meanwhile I disabled tests in snoopy-install.sh, with objective to reenable it when:
1. there is a flag for snoopy-install that enables skipping test suite,
2. or `make check` supports a flag that disables this particular test.

Personally I would prefer second option. As much as I hate it, it is far better that 99% of snoopy is tested before installing compared to 0% if --skip-all-tests is available.

Thanks for reporting, pushed, new snoopy-install.sh is already available.

PS: If you want to give it a shot, you can fix the install script to omit this particular test :)

**Comment by bostjan** [comment]
Fixed in 2.4.4.
